app/build.gradle

```diff
@@ -32,28 +33,17 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            shrinkResources false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
Does this change anything? AFAIK proguard is just disabled

Reply:
No, it doesn't. Proguard is still disabled, just that ```'proguard-android-optimize.txt'``` is a more aggressive optimization script than the regular ```'proguard-android.txt'```, which we don't need because Proguard is disabled anyways.